docs/foundation/component-variables/README.md

```diff
@@ -174,10 +173,11 @@ The naming structure for component variables and mixins is as follows:
 
 <br>
 
-## Getting Started
+## Getting Started (Comment: "Resources" is a better section title here)
 
 For more information on installing and using component variables in your project, view the [README.md on GitHub](https://github.com/rei/rei-cedar-component-variables). Additional examples and a list of supported components are located on the [cedar-component-variables doc site](
 https://rei.github.io/rei-cedar-component-variables/#/).  There is also a [CodeSandbox](https://codesandbox.io/s/qkwn78nw99) set up for testing out the component variables.
 
 
 </cdr-doc-table-of-contents-shell>
+(Comment: I don't see the 'How to use component variables' section from the Google doc here. Did that change?) 
```

Comment:
all that information is in the package readme on github/npm, i think it makes more sense to have that there than on the doc site: https://github.com/rei/rei-cedar-component-variables

Reply:
OK